src/main/java/rx/Observable.java

```diff
@@ -6705,6 +6706,26 @@ public final Boolean call(T t) {
     }
 
     /**
+     * Allow the an external signal control the amount of data being set through this Observable chain.
+     * When the control Observable emits false (closes the valve) requests upstream are stopped and any
+     * requests from downstream for more data are buffered until the control Observable emits a true
```

Comment:
Ah, okay. First I thought this operator always requests granularity no matter how many items the downstream requests but its about holding off requests by aggregating them if the valve is closed.